<?xml version="1.0" ?>
<testsuite errors="1" failures="0" name="TestDowntimes-20190515181741" tests="5" time="0.183">
	<testcase classname="TestDowntimes" name="test_entryonlydowntime" time="0.013"/>
	<testcase classname="TestDowntimes" name="test_setdelayedperiod" time="0.010"/>
	<testcase classname="TestDowntimes" name="test_setfrontendsecclass" time="0.019"/>
	<testcase classname="TestDowntimes" name="test_setperiodwithendtime" time="0.008"/>
	<testcase classname="TestDowntimes" name="test_downtimesfile" time="0.133">
		<error message="True is not false" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "./test_glideFactoryDowntimeLib.py", line 62, in test_downtimesfile
    check_time=now + 1))
AssertionError: True is not false
]]>		</error>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
